deploy/general-deploy/README.md

```diff
@@ -31,59 +26,23 @@ Create a file called `config` in the `/etc/kubeturbo/` directory, with the follo
 	"targetConfig": {
 		"probeCategory":"CloudNative",
 		"targetType":"Kubernetes",
-		"address":"<KUBERNETES_MASTER_ADDRESS>",
-		"username":"<KUBERNETES_USERNAME>",
-		"password":"<KUBERNETES_PASSWORD>"
+		"address":"http://<K8S_MASTER_IP>:8080",
+		"username":"user1",
+		"password":"pass1"
 	}
 }
 ```
-you can find an example with values [here](../config).
-
 
-### Step Two: Creating the Kubeturbo Static Pod
+### Step Two: Creating Kubeturbo Pod
 
 > NOTE: Ensure that you have completed Step One.
 
-Static pods are created by Kubelet. Based on whether kubeconfig is used, there are two ways to define the Kubeturbo pod template.
-Copy the Kubeturbo yaml pod definition to the configuration path used by Kubelet master nodes on startup. Typically, `/etc/kubernetes/manifests/`.
-
->NOTE: If api-server runs on localhost, `hostNetwork:true` must be added to pod spec
-
-#### Kubeturbo Pod Definition without kubeconfig
+Make sure you have `kubeconfig` and `config` under `/etc/kubeturbo`. Please copy them to `/etc/kubeturbo` on the node Kubeturbo will be running on. You can label the node as following to make sure Kubeturbo will be deployed on that node.
 
-```yaml
-apiVersion: v1
-kind: Pod
-metadata:
-  name: kubeturbo
-  labels:
-    name: kubeturbo
-spec:
-#  uncomment the following line if api server runs on http://127.0.0.1:8080
-#  hostNetwork: true
-  containers:
-  - name: kubeturbo
-    image: vmturbo/kubeturbo:<VERSION>
-    command:
-      - /bin/kubeturbo
-    args:
-      - --v=2
-      - --master=<Kubernetes_API_Server_URL>
-      - --turboconfig=/etc/kubeturbo/config
-    volumeMounts:
-    - name: turbo-config
-      mountPath: /etc/kubeturbo
-      readOnly: true
-  volumes:
-  - name: turbo-config
-    hostPath:
-      path: <DIRECTORY_CONTAINS_KUBETURBO_CONFIG_IN_HOST>
-  restartPolicy: Always
+```console
```

Comment:
Feel more comfortable to put this step at the very beginning:  Step 0, let the user choose a node, and label the node.  Then, user go to that node to prepare the files.